src/include/gurt/common.h

```diff
@@ -94,12 +94,15 @@ extern "C" {
 		if ((cond) && (ptr) != NULL) {				\
 			if (count <= 1)					\
 				D_DEBUG(DB_MEM,				\
-					"alloc(" #func ") '" name "': %i at %p.\n", \
+					"alloc(" #func ") '" name "':   \
+					                  %i at %p.\n", \
 					(int)(size), (ptr));		\
 			else						\
 				D_DEBUG(DB_MEM,				\
-					"alloc(" #func ") '" name "': %i * '" cname "':%i at %p.\n", \
-					(int)(size), (int)(count), (ptr)); \
+					"alloc(" #func ") '" name "': %i\
+					* '" cname "':%i at %p.\n",     \
```

Comment:
(style)  Avoid unnecessary line continuations
(style)  Avoid line continuations in quoted strings